bot/exts/recruitment/talentpool/_review.py

```diff
@@ -97,7 +98,12 @@ async def post_review(self, user_id: int, update_database: bool) -> None:
         thread = await last_message.create_thread(
             name=f"Nomination - {nominee}",
         )
-        await thread.send(fr"<@&{Roles.mod_team}> <@&{Roles.admins}>")
+        message = await thread.send(f"<@&{Roles.mod_team}> <@&{Roles.admins}>")
+
+        bump_cog: ThreadBumper = self.bot.get_cog("ThreadBumper")
+        if bump_cog:
+            context = await self.bot.get_context(message)
+            await bump_cog.add_thread_to_bump_list(context, thread)
```

Comment:
Not a fan of cogs depending on other cogs, but it's fine as a tentative solution.